packages/grid/_modules_/grid/components/GridVirtualScroller.tsx

```diff
@@ -6,7 +6,6 @@ import { unstable_composeClasses as composeClasses } from '@mui/material';
 import { useGridRootProps } from '../hooks/utils/useGridRootProps';
 import { useGridApiContext } from '../hooks/utils/useGridApiContext';
 import { useGridSelector } from '../hooks/utils/useGridSelector';
-import { gridScrollBarSizeSelector } from '../hooks/features/container/gridContainerSizesSelector';
```

Comment:
This should be a breaking change, no? The `gridContainerSizesSelector` is removed.

Reply:
https://github.com/mui-org/material-ui-x/pull/3002

Once this one is merged it will be private so no breaking change :)